lib/chef/resource/windows_task.rb

```diff
@@ -92,8 +92,9 @@ def after_created
 
       # Validate the passed value is numeric values only if it is a string
       def numeric_value_in_string?(val)
-        return false if val.is_a?(String) && /\D/ =~ val # \D is any non-numeric value
-        true
+        return true if Integer(val)
```

Comment:
if i were being super picky i'd note that `return` there is implicit :)

Reply:
I always feel a bit awful returning a non Boolean for a ? method. Totally works, but feels wrong at the same time.

Reply:
ah, i just meant `true if Integer(val)`